compiler/rustc_error_messages/locales/en-US/query_system.ftl

```diff
@@ -12,6 +12,8 @@ query_system_cycle_usage = cycle used when {$usage}
 
 query_system_cycle_stack_single = ...which immediately requires {$stack_bottom} again
 
+query_system_cycle_stack_middle = ...which requires {$desc}...
```

Comment:
@davidtwco How about those English words inside ($desc}?

`-	note: ...which requires computing the super traits of B...`

Reply:
That's a good point - these will need fixed, they're currently provided by the query system's macro  for defining queries, so that will require some future work to address.